api/controllers/user.js

```diff
@@ -247,7 +249,19 @@ async function listUser(req, res) {
 
 function removeUser(req, res) {
```

Comment:
You are not using this function on the feature.  Isnt it?

Reply:
Yes. These changes are unnecessary now.